src/libcore/str/mod.rs

```diff
@@ -174,10 +174,10 @@ impl Utf8Error {
 ///
 /// If you are sure that the byte slice is valid UTF-8, and you don't want to
 /// incur the overhead of the validity check, there is an unsafe version of
-/// this function, [`from_utf8_unchecked()`][fromutf8u], which has the same
-/// behavior but skips the check.
+/// this function, [`from_utf8_unchecked()`], which has the same behavior but
+/// skips the check.
 ///
-/// [fromutf8u]: fn.from_utf8_unchecked.html
+/// [`from_utf8_unchecked()`]: #method.from_utf8_unchecked
```

Comment:
I guess what I'm getting at is won't this link to this page: http://doc.rust-lang.org/nightly/std/str/fn.from_utf8.html#method.from_utf8_unchecked ? That anchor doesn't exist?